### PR TITLE
update versions, fix checks

### PR DESCRIPTION
**Pull request by Andranik999**

### CODE REVIEW

File: apps/expo/package.json [hunk 1]
```diff
@@ -37,12 +37,11 @@
     "expo-status-bar": "~1.12.1",
     "expo-web-browser": "^13.0.3",
     "nativewind": "~4.0.36",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
+    "react": "catalog:react18",
+    "react-dom": "catalog:react18",
```
Comment (nadavl) [inline apps/expo/package.json:41]: Is it recommened / need to use catalog?

Comment (nadavl) [reply apps/expo/package.json:41]: I see it is, but then the version should only be defined in a single place and other places should just use "catalog:", no?

https://www.perplexity.ai/search/explain-pnpm-catalog-sAfgcBjjSYus7iiUNgt_bA

**Review by nadavl — COMMENTED** [review]
Great, see comment

File: apps/expo/package.json (hunk 1 above)
Comment (Andranik999) [reply apps/expo/package.json:41]: Yes, it is defined in workspace.yaml and used in other places

**Review by nadavl — APPROVED** [review]

**Pull request merged**